### PR TITLE
cut 0.19.0

### PR DESCRIPTION
**Pull request by Nexucis**
Signed-off-by: Augustin Husson <husson.augustin@gmail.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+0.19.0 / 2021-05-19
+===================
+
+* **[Feature]**: Provide an alternative `MetricName` top-level entrypoint
+* **[Enhancement]**: Improve the way to manage the number, and the signed number
+* **[BugFix]**: Fix parsing metric beginning by `InF`/`NaN` like `infra`
```
Comment (juliusv) [inline CHANGELOG.md]:
```suggestion
* **[BugFix]**: Fix parsing metric names starting with `Inf`/`NaN` like `infra`
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
+0.19.0 / 2021-05-19
+===================
+
+* **[Feature]**: Provide an alternative `MetricName` top-level entrypoint
+* **[Enhancement]**: Improve the way to manage the number, and the signed number
```
Comment (juliusv) [inline CHANGELOG.md]: I'd just say

```suggestion
* **[Enhancement]**: Improve parsing of numbers
```

**Comment by Nexucis** [comment]
all fixed @juliusv 👍

**Comment by juliusv** [comment]
:+1:

**Pull request merged**